### PR TITLE
chore(test): add travis-ci config

### PR DESCRIPTION
**Pull request by pgoldrbx**
Moving this library to Travis CI to centralize more of our tooling.

https://travis-ci.com/CondeNast/copilot-util

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+cache: npm
+
+node_js:
+  - "6"
+  - "8"
+  - "10"
+
+branches:
+  only:
+    - master
```
Comment (tollmanz) [inline .travis.yml]: I can never remember, does this block pull requests builds?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+notifications:
+  email: false
+
+before_install:
+  - npm config set //registry.npmjs.org/:_authToken=$NPM_TOKEN
```
Comment (tollmanz) [inline .travis.yml]: Reminder to add this env var when you set up the repo on Travis.

File: .travis.yml (hunk 1 above)
Comment (pgoldrbx) [reply .travis.yml]: There are two types of builds we can do: branch builds, or PR builds. If we don't set this, and we allow both in the UI, then our PRs will trigger both builds at once (duplicate). By setting this, we only run builds on pushes to master or on PRs that are opened. We don't trigger builds on pushed branches that are not used to open a PR.

File: .travis.yml (hunk 2 above)
Comment (pgoldrbx) [reply .travis.yml]: yep. it was done

File: .travis.yml (hunk 1 above)
Comment (tollmanz) [reply .travis.yml]: That's not confusing at all

**Pull request merged**